### DISCUSSION TITLE
How do I get the pull request information<title>

### DISCUSSION THREAD

**Discussion by sbalakrishn** [community/community#47331] [Apps, API and Webhooks]
### Select Topic Area

Product Feedback

### Body

owner=###
repo=###
pull_request=14
gh api -H "Accept: application/vnd.github+json" repos/${owner}/8x8/${repo}/pulls/${pull_request}

The above is giving an error :
{
  "message": "Not Found",
  "documentation_url": "https://docs.github.com/rest/reference/pulls#list-pull-requests"
}
gh: Not Found (HTTP 404)

**Comment by jge162** [comment]
you need to double check the repository name, and pull request number. Also assuming it is your repo then there is an issue with pull number or name.

**Reply by jge162** [reply]
also are you removing the curly braces? 

`gh api -H "Accept: application/vnd.github+json" repos/sbalakrishn/auto_gitops_dns_test_srj/pulls/14`

**Reply by sbalakrishn** [reply]
@jge162 , thanks for quick response , I will check those areas and update you

**Reply by sbalakrishn** [reply]
I tried removing the curly braces

[root@atlantis-test-cc ~]# owner=####
[root@atlantis-test-cc ~]# repo=8x8/auto_gitops_dns_test_srj
[root@atlantis-test-cc ~]# pull_request=14
[root@atlantis-test-cc ~]# gh api -H "Accept: application/vnd.github+json" repos/owner/8x8/repo/pulls/pull_request
{
  "message": "Not Found",
  "documentation_url": "https://docs.github.com/rest"
}
gh: Not Found (HTTP 404)

**Comment by sbalakrishn** [comment]
I tried with the curly braces

owner=sbalakrishn
[root@atlantis-test-cc ~]# repo=8x8/auto_gitops_dns_test_srj
[root@atlantis-test-cc ~]# pull_request=14
[root@atlantis-test-cc ~]# gh api -H "Accept: application/vnd.github+json" repos/${owner}/8x8/${repo}/pulls/${pull_request}
{
  "message": "Not Found",
  "documentation_url": "https://docs.github.com/rest"
}
gh: Not Found (HTTP 404)

**Reply by sbalakrishn** [reply]
do you find anything wrong with the command here ? I don't know what is causing this error

**Reply by jge162** [reply]
I do not see any mistakes in syntax, try this -> `gh api -H "Authorization: token <your_token>" -H "Accept: application/vnd.github+json" repos/${owner}/8x8/${repo}/pulls/${pull_request}`

**Reply by jge162** [reply]
oh have you tried Rest Api directly? `curl -H "Accept: application/vnd.github+json" https://api.github.com/repos/${owner}/8x8/${repo}/pulls/${pull_request}`

**Reply by sbalakrishn** [reply]
let me try that

**Reply by sbalakrishn** [reply]
[root@atlantis-test-cc ~]# export owner=###
[root@atlantis-test-cc ~]# export repo=8x8/auto_gitops_dns_test_srj
[root@atlantis-test-cc ~]# gh api -H "Authorization: token #####" -H "Accept: application/vnd.github+json" repos/${owner}/8x8/${repo}/pulls/${pull_request}
{
  "message": "Not Found",
  "documentation_url": "https://docs.github.com/rest"
}
gh: Not Found (HTTP 404)

**Reply by jge162** [reply]
forgot a / try again sorry `gh api -H "Authorization: token <your_token>" -H "Accept: application/vnd.github+json" /repos/${owner}/8x8/${repo}/pulls/${pull_request}`

**Reply by jge162** [reply]
if this does not work then IDK

**Reply by jge162** [reply]
you are sure that Pull request exits and no typos in credentials?

**Reply by sbalakrishn** [reply]
export owner=###
[root@atlantis-test-cc ~]# export repo=8x8/auto_gitops_dns_test_srj
[root@atlantis-test-cc ~]# export pull_request=14
[root@atlantis-test-cc ~]# gh api -H "Authorization: token #####" -H "Accept: application/vnd.github+json" repos/${owner}/8x8/${repo}/pulls/${pull_request}
{
  "message": "Not Found",
  "documentation_url": "https://docs.github.com/rest"
}
gh: Not Found (HTTP 404)

**Reply by sbalakrishn** [reply]
Yes the pull request exist and not credential typos indeed

**Reply by jge162** [reply]
try this `curl -v -H "Authorization: token <your_token>" -H "Accept: application/vnd.github+json" https://api.github.com/repos/${owner}/8x8/${repo}/pulls/${pull_request}`

**Reply by jge162** [reply]
if this doesn't work you need [GitHub Support](https://support.github.com/contact?tags=hubberfy_account%2Crr-accounts) create a ticket for help.

**Reply by sbalakrishn** [reply]
Thanks @jge162

**Reply by ghostinhershell** [reply]
@sbalakrishn just wanted to follow-up here, did @jge162's feedback answer your question here?
